NOTICE

```diff
@@ -65,4 +68,7 @@ distribution of the software in this repository outside of Code.org's own
 website may not include any use of this proprietary IP without a separate
 agreement with DreamWorks Animation LLC.
 
-Code.org uses a set of Font Awesome proprietary icons, related computer software, and other material published at github.com/FortAwesome/Font-Awesome and fontawesome.com licensed under the Font Awesome Pro License (https://fontawesome.com/license).
+Code.org uses a set of Font Awesome proprietary icons, related computer
+software, and other material published at github.com/FortAwesome/Font-Awesome
+and fontawesome.com licensed under the Font Awesome Pro License
+(https://fontawesome.com/license).
```

Comment:
This is just updating the formatting so the text wraps with the rest of the paragraph